routes/index.js

```diff
@@ -579,3 +658,14 @@ function checkRegistration(req, res, next) {
     next();
   }
 }
+
```

Comment:
I was thinking checkRegistration might be able to call this as a helper function above; I couldn't use that function directly because in my case, I always want to preserve the data captured. checkRegistration only preserves data if/when there are errors. I'd like to worry about that later though, if possible so we can get this functionality included sooner rather than later.